.github/workflows/push-to-stage.yml

```diff
@@ -0,0 +1,44 @@
+name: Push to Staging
+
+on: 
+  push:
+  pull_request:
+    types:
+      - opened
```

Comment:
Will this rerun if I push new commits to my fork branch which already has an open PR?

Reply:
The pull request event will only fire when the PR is opened. In this workflow we also fire on push to update the config files. The pr opened event is mainly listened for to add the comment with the config url.